source/common/stream_info/filter_state_impl.cc

```diff
@@ -7,12 +7,12 @@ namespace StreamInfo {
 
 void FilterStateImpl::setData(absl::string_view data_name, std::unique_ptr<Object>&& data) {
   // TODO(Google): Remove string conversion when fixed internally.
+  // absl::string_view will not convert to std::string without an explicit cast; see
+  // https://github.com/abseil/abseil-cpp/blob/master/absl/strings/string_view.h#L328
```

Comment:
Request: Could you rewrite this as "Fixing this TODO will also require an explicit cast from absl::string_view to std::string in the data_storage_ index below; see <URL>".  I think it's confusing to interpose it between the TODO and the line the TODO is describing without creating a linkage, but it was confusing where it was before as well, so just combining them seems the best idea.